pandas/tests/frame/test_query_eval.py

```diff
@@ -870,7 +880,7 @@ def test_str_query_method(self, parser, engine):
 
             for lhs, op, rhs in zip(lhs, ops, rhs):
                 ex = f"{lhs} {op} {rhs}"
-                msg = r"'(Not)?In' nodes are not implemented"
+                # msg = r"'(Not)?In' nodes are not implemented"
```

Comment:
don't leave this commented out. out of curiosity why move outside if block if message repeated on L911?

Reply:
I left that in there by mistake. Also, I overlooked the if block. I'll put it there. (I had originally put it inside of the for loop)

Reply:
can you remove this line entirely.

Reply:
Yeah, I removed it:
https://github.com/pandas-dev/pandas/pull/34940/commits/14dd0b3c49ce161d4af237d6d40afee6775a538a

Reply:
still showing in diff.  we don't want commented out code left behind.

Reply:
whoops.. my bad. test_str_query_method and test_str_LIST_query_method got me confused. I've updated it